INSTALL

```diff
@@ -0,0 +1,368 @@
+Installation Instructions
+*************************
+
+   Copyright (C) 1994-1996, 1999-2002, 2004-2016 Free Software
+Foundation, Inc.
+
+   Copying and distribution of this file, with or without modification,
+are permitted in any medium without royalty provided the copyright
+notice and this notice are preserved.  This file is offered as-is,
+without warranty of any kind.
+
+Basic Installation
+==================
+
+   Briefly, the shell command './configure && make && make install'
```

Comment:
@msalle: Didn't notice this until I tried to build UberFTP manually on openSUSE Leap 15.1: `configure` is removed with this patch set, only `configure.ac` remains. I think this is similar to the GCT, but then this INSTALL document is missing the step to actually create `configure` (e.g. `autoreconf -i`?).

Reply:
This is the normal setup with the autotools in my opinion: you don't want to include the output of the autotools in the git or svn repository, but you do want to have an INSTALL file, which will end up in the output of "make dist", which is what you're distributing. If you leave out the INSTALL file, some of the autotools complain.
We could add an `autogen.sh` or `bootstrap` script, similar to those of e.g. [VOMS](https://github.com/italiangrid/voms/blob/master/autogen.sh) or [LCMAPS](https://ndpfsvn.nikhef.nl/viewvc/mwsec/trunk/lcmaps/bootstrap?view=markup)

Reply:
Ok, I agree, though I seem to remember seeing other software distributing a ready-made `configure` script, but these possibly don't use autotools. [The autotools FAQ](https://www.gnu.org/software/automake/faq/autotools-faq.html#What-does-_002e_002fbootstrap-or-_002e_002fautogen_002esh-do_003f) recommends to use `bootstrap` over `autogen.sh`. For me `autoreconf -i` was enough to get things ready for the configure run. Do we need anything in addition in this file?

Reply:
Probably not, `autoreconf -i` is basically doing all those steps (I would then add the `-v` as in the FAQ you quoted). That's also why many people don't bother shipping a `bootstrap` file I think.

Reply:
Ok, looks like I cannot create another commit for this PR to add a `bootstrap`- most likely because I created it using a branch from your fork instead of letting you create it from your fork - otherwise it could allow me to make changes in any remote branch on GitHub I take as base for a PR. I'd propose the following contents for `bootstrap`:

```
#!/bin/sh
autoreconf -vi
```

...if you like also with `space` between shebang and path, but I prefer it w/o.

Reply:
I've just added a bootstrap.

Reply:
Ok, great.